.ci/Jenkinsfile

```diff
@@ -245,8 +245,7 @@ pipeline {
   }
   post {
     cleanup {
-      notifyBuildResult(prComment: true, slackComment: true, analyzeFlakey: true,
-                        flakyReportIdx: 'reporter-apm-shared-apm-pipeline-library-master')
+      notifyBuildResult(prComment: true, slackComment: true, analyzeFlakey: true, jobName: getFlakyJobName(withBranch: 'master'))
```

Comment:
For instance:

the jobName for our apm-pipeline-library will be : `apm-shared/apm-pipeline-library-mbp/master`

Reply:
Love this!